### ISSUE TITLE
Admin Panel

### ISSUE THREAD

**Issue by eyeamkd** [scale-jobs/abfy.io#5]
Create an Admin Panel 
<Insert Wireframes >

**Comment by eyeamkd** [comment]
1. Login and see your Experiment Results 
2. Get your API Key
3. Render Stats
4. KeyAction based Results Display
5. Combination of variants per experiment

**Comment by balajikummari** [comment]
![image](https://github.com/scale-jobs/abfy.io/assets/45452664/fdeaccbf-efb7-4c92-8f0f-729d8858c51d)

Let's try to stick to a very simple setup for now 

- First time user (No JWT), generate  a API Keys : Public key(goes into package) and Private key
-  use public key in package
- Paste Private key into UI for Login, gen JWT and bakend and store in cookie or somewhere
- logout button clears the jwt

![image](https://github.com/scale-jobs/abfy.io/assets/45452664/e212b82d-af26-40d6-bc3e-93e3f45a0004)

![image](https://github.com/scale-jobs/abfy.io/assets/45452664/ce2c3224-46c2-4190-aff8-7e3e1385573e)

**Comment by eyeamkd** [comment]
Expecting the users to save a private key for logging in, doesn't sink in with me. 
Instead, let's just have a hassle free OAuth experience for signing in, let the users create mulitple apps. 
For each app we can give them a dedicated API Key. 

Why this approach: 
1. Would make things simpler 
2. No pressure on the user to save a key or any info to login 
3. Follows the standard of creating apps ( Gtag manager ) etc. So users would be pre-trained to use something like this

**Comment by eyeamkd** [comment]
We can go with Mongo for storing experiment results. I'm skeptical about using postgres for auth info or API key info, as it would create a mix of SQL/ NoSQL, but lowkey want to try this setup, best of both worlds.  Your take @balajikummari?

**Comment by balajikummari** [comment]
https://www.prisma.io/docs/orm/overview/databases/mongodb

With prisma you can have both, let the user decide what he wants

**Comment by balajikummari** [comment]
https://www.prisma.io/docs/orm/overview/databases

All supported DBs

**Comment by eyeamkd** [comment]
Yeah ORM was never an issue.

**Comment by balajikummari** [comment]
I mean as long as you are using prisma, we are good with any Db you use

**Comment by eyeamkd** [comment]
Coool. Let's use Prisma only then. What about OAuth and a single API Key approach?

**Comment by balajikummari** [comment]
Lets use simple API key or jwt auth,
oAuth and all overkill for now ?

Simply sign a jwt/generate keys and send to FE and verify JWT/keys on
backend?

**Comment by eyeamkd** [comment]
I agree with your point of overkill, but I don't like the idea of user having to store/save something on his end. 
On the other hand, OAuth would only go about for being a fundamental base in our architecture that we don't need to change

**Comment by eyeamkd** [comment]
Yo @balajikummari given that the weekend is near, imma start with this then. Just wanted to give you an FYI
